### ISSUE TITLE
Preview doesn't show up for the first backend

### ISSUE THREAD

**Issue by na2hiro** [LouisBrunner/dnd-multi-backend#46]
Preview doesn't seem to work if a backend is the first one in the pipeline.

I created a custom pipeline combining mouse backend and touch backend, both of which don't have a default preview:

```tsx
import MouseBackend from 'react-dnd-mouse-backend';
import TouchBackend from 'react-dnd-touch-backend';
import { TouchTransition, MouseTransition } from 'react-dnd-multi-backend';

const MouseToTouch = {
    backends: [
        {
            backend: MouseBackend,
            preview: true,
            transition: MouseTransition
        },
        {
            backend: TouchBackend,
            options: {enableMouseEvents: true},
            preview: true,
            transition: TouchTransition
        },
    ]
};
export default MouseToTouch;
```

I implemented a preview and I can see preview on touch device, but cannot see with mouse devices. Interestingly, if I flip the order of the backends, then it's vice versa: preview shows up for mouse devices, but stops to show up for touch devices.

Could you check if it's a bug? Using hooks, context nor HOC doesn't seem to matter.

**Comment by LouisBrunner** [comment]
Hi @na2hiro,

Thanks for your report! Could you clarify which version you are using and if using the lastest (`6.0.0`) or an earlier one (`5.0.0`) changes anything?

**Comment by na2hiro** [comment]
Hi, I confirmed it with 6.0.0. I also tried with 5.0.0 but I think I only checked the Mouse -> Touch order (in which case preview didn't work for mouse)

**Comment by LouisBrunner** [comment]
Thanks for the confirmation. I think I have isolated the bug, I have just released a new version including the fix: `6.0.1`. Can you confirm that it solves your issue?

**Comment by na2hiro** [comment]
I confirmed that both order works for both devices. Thanks for the very quick response!!
